contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.1.0--3.2.0.sql

```diff
@@ -108,6 +108,30 @@ CREATE AGGREGATE sys.VARP(float8) (
 CREATE OR REPLACE FUNCTION sys.rowcount_big()
 RETURNS BIGINT AS 'babelfishpg_tsql' LANGUAGE C STABLE;
 
+CREATE OR REPLACE FUNCTION sys.database_principal_id(IN user_name sys.sysname)
+RETURNS OID
+AS 'babelfishpg_tsql', 'user_id'
+LANGUAGE C IMMUTABLE PARALLEL SAFE STRICT;
+
+CREATE OR REPLACE FUNCTION sys.database_principal_id()
+RETURNS OID
+AS 'babelfishpg_tsql', 'user_id_noarg'
+LANGUAGE C IMMUTABLE PARALLEL SAFE;
+
+ALTER FUNCTION sys.user_id(IN user_name TEXT DEFAULT NULL) RENAME TO babelfish_sys_user_id_deprecated_in_3_2_0;
```

Comment:
DEFAULT is not accepted in PG ALTER FUNCTION syntax. See https://www.postgresql.org/docs/current/sql-alterfunction.html. And we should just add the deprecated string after the function name to avoid confusion.
`ALTER FUNCTION sys.user_id(TEXT) RENAME TO user_id_deprecated_in_3_2_0;`